modules/cnv-creating-template-wizard-web.adoc

```diff
@@ -5,32 +5,31 @@
 [id="cnv-creating-template-wizard-web_{context}"]
 = Creating a virtual machine template with the interactive wizard in the web console
 
-The web console features an interactive wizard that guides you through the *Basic Settings*, 
-*Networking*, and *Storage* screens to simplify the process of creating virtual machine templates. 
-All required fields are marked with a `*`. The wizard prevents you from moving to the next screen 
+The web console features an interactive wizard that guides you through the *General*,
+*Networking*, *Storage*, *Advanced*, and *Review* steps to simplify the process of creating virtual machine templates.
+All required fields are marked with a `*`. The wizard prevents you from moving to the next step
 until you provide values in the required fields.
 
 .Procedure
 
 . In the {CNVProductName} console, click *Workloads* -> *Virtual Machine Templates*.
-. Click *Create Template* and select *Create with Wizard*. 
-. Fill in all required *Basic Settings*. 
-. Click *Next* to progress to the *Networking* screen. A NIC that is named `nic0` is attached by default. 
-.. Optional: Click *Create NIC* to create additional NICs. 
-.. Optional: You can remove any or all NICs by clicking the Options menu {kebab} and selecting *Remove NIC*. Virtual machines created from a template do not need a NIC attached. NICs can be created after a virtual machine has been created. 
+. Click *Create Template* and select *New with the Wizard*.
+. Fill in all required fields in the *General* step.
+. Click *Next* to progress to the *Networking* screen. A NIC that is named `nic0` is attached by default.
+.. Optional: Click *Add Network interface* to create additional NICs.
```

Comment:
Should this be capitalized `Add Network Interface`?

Reply:
The UI design shows "Add Network interface". However, the correct capitalization as you indicated should be "Add Network Interface." Will track this as a defect with UI to ensure that the UI displays correct capitalization.